### ISSUE TITLE
Grid lines are only drawn inside the currently-seen portion of the page

### ISSUE THREAD

**Issue by jensimmons** [mozilla/css-grid-inspector#26]
I'm seeing a problem where the Grid Inspector only renders the lines in the currently-seen viewport. Then when I scroll, I see more page where the grid lines are missing.

**Comment by potch** [comment]
Do the grid lines appear to "scroll" (or otherwise change) in the area where they do render? or do they only render once and remain static?

**Comment by jensimmons** [comment]
No, the lines didn't change — they were drawn over top of the part of the page that was visible. And as that page was slid upward, they slide with the section they'd been drawn on. Then the part of the page that was originally hidden, as it came into view, was without any lines. I started out in RWD mode, and thought at first maybe the bug was only there. Then I switched out of the mode, toggled the tool on and off... problem remained. Then I refreshed. Problem was still happening. Then I refreshed multiple times and it seemed to go away.
